src/components/input-group/daikin-input-group.ts

```diff
@@ -125,19 +144,38 @@ export class DaikinInputGroup extends LitElement {
   error = "";
 
   /**
-   * Whether to display the counter in the Textarea
+   * Maximum value to display on the counter. When `null`, the counter will be hidden.
    */
-  @property({ type: Boolean, reflect: true })
-  textareaCounter = false;
+  @property({ type: Number, reflect: true, attribute: "textarea-max-count" })
+  textareaMaxCount: number | null = null;
 
   @queryAssignedElements({ selector: "daikin-textarea" })
   _textareas!: DaikinTextarea[];
 
   @queryAssignedElements({ selector: "daikin-text-input,daikin-textarea" })
   _controls!: ControlElement[];
 
+  @state()
+  _hasTextarea = false;
+
+  @state()
+  _textareaValueLength = 0;
```

Comment:
Let's combine the states into one.
```suggestion
  @state()
  private _textareaValueLength: number | null = null;
```